plutus-ledger-constraints/src/Ledger/Constraints/OnChain/V2.hs

```diff
@@ -139,7 +140,7 @@ checkTxConstraint ctx@ScriptContext{scriptContextTxInfo} = \case
                    Ada.fromValue txOutValue >= Ada.fromValue vl
                 && Ada.fromValue txOutValue <= Ada.fromValue vl + Ledger.maxMinAdaTxOut
                 && Value.noAdaValue txOutValue == Value.noAdaValue vl
-                && dv == id
+                && getOutDatum dv == id -- FIXME
```

Comment:
Add a comment with the problem?

Reply:
I thought I get rid of the "fixme", sorry for it. It might not be an issue, I used it while coding, to be sure that I would pass on it again in the end and check if it should be changed, I'll take a look at it.